explainer.html

```diff
@@ -225,6 +225,29 @@ <h2>Defining an RDF Dataset Hash and/or Signing RDF Datasets</h2>
     </div>
   </section>
 
+  <section id="separation">
+    <h2>Separation of Concepts in the Deliverables</h2>
```

Comment:
didn't you mean 
```suggestion
    <h2>Separation of Concerns in the Deliverables</h2>
```
?

Reply:
Actually, I meant concepts...